src/libvirtApi/domain.js

```diff
@@ -112,8 +112,9 @@ function domainAttachDevice({ connectionName, vmId, permanent, hotplug, xmlDesc
     return call(connectionName, vmId, 'org.libvirt.Domain', 'AttachDevice', [xmlDesc, flags], { timeout, type: 'su' });
 }
 
-export function getPythonPath() {
-    return cockpit.spawn(["/bin/sh", "-c", "command -v /usr/libexec/platform-python || command -v python3 || command -v python"]).then(pyexe => { pythonPath = pyexe.trim() });
+export async function getPythonPath() {
+    const out = await cockpit.spawn(["/bin/sh", "-c", "command -v /usr/libexec/platform-python || command -v python3 || command -v python"]);
+    pythonPath = out.trim();
 }
```

Comment:
Side note, should this not use `pkg/lib/python.js`?

Reply:
It can't at the moment but I guess expanding it to be possible would be nice, but can't say if that can be done without API breakage.

Reply:
Right, we can add such a thing to python.js and try to make it use that. I'd like to keep this PR as strictly syntactical, but I made a TODO item for this (this indeed bothered me as well).

Reply:
Something for a :cloud_with_rain: day indeed